pkg/sinit/writer.go

```diff
@@ -0,0 +1,21 @@
+package sinit
+
+import (
+	"errors"
+	"log"
+)
+
+type logLoggerWriter struct {
+	dest *log.Logger
+}
+
+var ErrLoggerNotInitialized = errors.New("log logger is not initialized")
+
+func (w *logLoggerWriter) Write(p []byte) (n int, err error) {
+	if w.dest == nil {
+		return 0, ErrLoggerNotInitialized
+	}
+
+	w.dest.Output(2, string(p))
```

Comment:
I think the `2` here would benefit from a comment or just a var name, maybe `calldepth := 2` and then pass it (or a comment) otherwise its a bit hard to follow what the "2" means (is it a fd?) something else?

Reply:
And also a test for the call depth so it matches what is expected, I have seen a bug in this previously in other parts of this project so it is good to test this.